src/engine/ResultTable.h

```diff
@@ -29,28 +30,27 @@ class ResultTable {
 
   ResultTable();
 
-  ResultTable(const ResultTable&);
+  ResultTable(const ResultTable &);
```

Comment:
Shouldn't this be '(const ResultTable&)'? In the Result Table, the ampersand character is aligned with the variable, not the class, while in other classes it seems to be the other way around.

Reply:
I agree, normally I use clang-format with Google Style + left pointers (via a `.clang-format` file in the repository which was missing and so it used the default LLVM style, creating this inconsistency) but running this on QLever proposes A LOT of changes and it even breaks the build because of weird/broken inter-Header dependencies. See my formatting branch. I'm not sure if it makes sense to just fix this up once and be done with it or keep using some semi-Google Style and maybe doing clang-format per file. In the meantime I guess we could at least add the `.clang-format` file?

Reply:
Ok so here is my plan for this. I'll do another Pull Request just for adding the `.clang-format` File and then we apply it to every changed file in each new commit. I thought about doing it for this PR already but it touches 30 files and would obscure the actual changes too much. Maybe we have to decide on a case by case basis if we do an extra style commit then.